features/step_definitions/metering.rb

```diff
@@ -294,12 +296,21 @@
   step %Q/I switch to cluster admin pseudo user/ unless env.is_admin? user
   if project(metering_ns).exists?
     step %Q(I ensure "#{metering_ns}" meteringconfig is deleted)
+    # need to remove CRDs as well.
+    step %Q(I remove all metering CRDs with label "operators.coreos.com/metering-ocp.openshift-metering=")
+    step %Q(I ensure "openshift-metering" meteringconfig is deleted)
     step %Q(I ensure "metering-ocp-sub" subscription is deleted)
     step %Q(I ensure "metering-ocp-og" subscription is deleted)
     step %Q/I ensure "#{metering_ns}" project is deleted/
   end
 end
 
+Given /^I remove all metering CRDs with label #{QUOTED}$/ do | label |
```

Comment:
This seems to copy the other step. It implies it selects only metering CRDs. But in fact it has no special extra logic. IMO we should remove this step.